app/templates/app/update_installment.html

```diff
@@ -20,7 +20,7 @@ <h1>Update installment information</h1>
             </div>
         </div>
     </div>
-    <form method="post">{% csrf_token %}
+    <form action="" method="post">{% csrf_token %}
```

Comment:
Why action is blank?

Reply:
Trying to do something else, I made a mistake.